tests/ctst/world/Zenko.ts

```diff
@@ -213,20 +274,25 @@ export default class Zenko extends World {
             }
             // Getting account ID
             const account = await SuperAdmin.getAccount({
-                name: this.parameters.AccountName || Constants.ACCOUNT_NAME,
-            });
+                accountName: this.parameters.AccountName || Constants.ACCOUNT_NAME,
+            }) as Utils.AccountObject;
             // Getting roles with GetRolesForWebIdentity
-            const { data: { data: roles } } = await SuperAdmin.getRolesForWebIdentity(this.options.webIdentityToken);            // Get the first role with the storage-manager-role name
-            let roleToAssume = '';
-            if (roles.ListOfRoleArns) {
-                roleToAssume = roles.find(
-                    (roleArn: string) => roleArn.includes(ARWWITargetRole) && roleArn.includes(account.id),
+            // Get the first role with the storage-manager-role name
+            const data: GetRolesForWIResponse =
+                (await SuperAdmin.getRolesForWebIdentity(this.options.webIdentityToken)).data as GetRolesForWIResponse;
+            let roleToAssume: string | undefined = '';
+
+            // TODO see necessity of if check: from my understanding, and from what I've seen in both Vault projects
```

Comment:
to be removed maybe? I think this test is needed

Reply:
02d2a7109fdea473e90ed07623ee8b058e4ab026